docs/testing.md

```diff
@@ -0,0 +1,24 @@
+# Testing on Staging
+In general it is hard if not impossible to test these processes end to end before putting them into production.  
+You can do some testing on staging for the parts that run within the staging environment.  
+For example make sure that files get cretaed in the samba share as expected of that files get put onto the FTP server.
+Unfotunately when the system interacts with an API instead of a file based transfer this can, like the NCIP renewal, this can be almost impossible to test from staging. 
```

Comment:
```suggestion
  Unfortunately when the system interacts with an API instead of a file based transfer, like the NCIP renewal, this can be almost impossible to test from staging.
```